util/execdetails/execdetails_test.go

```
@@ -111,51 +105,39 @@ func TestCopRuntimeStats(t *testing.T) {
 		RocksdbBlockReadByte:      100,
 	}
 	stats.RecordScanDetail(tableScanID, "tikv", scanDetail)
-	if stats.ExistsCopStats(tableScanID) != true {
-		t.Fatal("exist")
-	}
+	assert.True(t, stats.ExistsCopStats(tableScanID))
+
 	cop := stats.GetOrCreateCopStats(tableScanID, "tikv")
-	if cop.String() != "tikv_task:{proc max:2ns, min:1ns, p80:2ns, p95:2ns, iters:3, tasks:2}, "+
-		"scan_detail: {total_process_keys: 10, total_keys: 15, rocksdb: {delete_skipped_count: 5, key_skipped_count: 1, block: {cache_hit_count: 10, read_count: 20, read_byte: 100 Bytes}}}" {
-		t.Fatalf(cop.String())
-	}
+	expected := "tikv_task:{proc max:2ns, min:1ns, p80:2ns, p95:2ns, iters:3, tasks:2}, " +
+		"scan_detail: {total_process_keys: 10, total_keys: 15, rocksdb: {delete_skipped_count: 5, key_skipped_count: 1, block: {cache_hit_count: 10, read_count: 20, read_byte: 100 Bytes}}}"
+	assert.Equal(t, expected, cop.String())
+
 	copStats := cop.stats["8.8.8.8"]
-	if copStats == nil {
-		t.Fatal("cop stats is nil")
-	}
+	assert.NotNil(t, copStats)
```

`require.NotNil` will fail faster and better.